tests/integration/targets/setup_connection_aws_ssm/defaults/main.yml

```diff
@@ -0,0 +1,46 @@
+---
+instance_type: t3.micro
+
+ami_details:
+    fedora:
+        owner: 125523088429
+        name: Fedora-Cloud-Base-34-1.2.x86_64*
+        user_data: |
+            #!/bin/sh
+            sudo dnf install -y https://s3.amazonaws.com/ec2-downloads-windows/SSMAgent/latest/linux_amd64/amazon-ssm-agent.rpm
+            sudo systemctl start amazon-ssm-agent
+        os_type: linux
+    amazon:
+        owner: amazon
+        name: amzn2-ami-kernel-5.10-hvm-*-x86_64-gp2
+        user_data: |
+            #!/bin/sh
+            # Pre-Installed ?
```

Comment:
>             # Pre-Installed ?
afaik yes

Reply:
Lol, yes it is, that was from when I was still getting the first rework written.  I'll remove the ?

Reply:
yes. let's reduce confusion

Reply:
fixed in my local copy, will push once test run finishes